orderBook.js

```diff
@@ -0,0 +1,39 @@
+const reconcileOrder = (existingBook, incomingOrder) => {
+
+  let updatedBook = [...existingBook]
```

Comment:
If you started with an empty array and filled it as you went you wouldn't have needed to change the order in the test